modules/ROOT/pages/tutorial-basic-provisioning.adoc

```diff
@@ -0,0 +1,268 @@
+tutorial
+----
+
+== Exploring Fedora CoreOS Internals
+
+Once you have access to the console of the machine you can browse around a bit to see some of the different pieces of the operating system.
+For example, even though this is an OSTree based system it was still composed via RPMs and you can inspect the system to see what it was composed of by using rpm:
+
+[source,bash]
+----
+[core@tutorial ~]$ rpm -q ignition kernel moby-engine podman systemd rpm-ostree zincati
+ignition-2.4.1-1.git5260a5b.fc32.x86_64
+kernel-5.7.8-200.fc32.x86_64
+moby-engine-19.03.11-1.ce.git42e35e6.fc32.x86_64
+podman-1.9.3-1.fc32.x86_64
+systemd-245.6-2.fc32.x86_64
+rpm-ostree-2020.3-1.fc32.x86_64
+zincati-0.0.12-2.fc32.x86_64
+----
+
+You can also inspect the current revision of Fedora CoreOS:
+
+[source,bash]
+----
+[core@tutorial ~]$ rpm-ostree status
+State: idle
+Deployments:
+* ostree://fedora:fedora/x86_64/coreos/stable
+                   Version: 32.20200715.3.0 (2020-07-27T11:36:29Z)
+                    Commit: a3b08ee51b1d950afd9d0d73f32d5424ad52c7703a6b5830e0dc11c3a682d869
+              GPGSignature: Valid signature by 97A1AE57C3A2372CCA3A4ABA6C13026D12C944D0
+----
+
+And check on zincati.service, which communicates with our update server and tells rpm-ostree when to do an update and to what version to update to:
+
+[source,bash]
+----
+[core@tutorial ~]$ systemctl status zincati.service | cat
+● zincati.service - Zincati Update Agent
+     Loaded: loaded (/usr/lib/systemd/system/zincati.service; enabled; vendor preset: enabled)
+     Active: active (running) since Thu 2020-08-06 14:50:36 UTC; 1h 41min ago
+       Docs: https://github.com/coreos/zincati
+   Main PID: 889 (zincati)
+      Tasks: 2 (limit: 2288)
+     Memory: 14.2M
+     CGroup: /system.slice/zincati.service
+             └─889 /usr/libexec/zincati agent -v
+
+Aug 06 14:50:36 tutorial systemd[1]: Started Zincati Update Agent.
+Aug 06 14:50:36 tutorial zincati[889]: [INFO ] starting update agent (zincati 0.0.12)
+Aug 06 14:50:39 tutorial zincati[889]: [INFO ] Cincinnati service: https://updates.coreos.fedoraproject.org
+Aug 06 14:50:39 tutorial zincati[889]: [INFO ] agent running on node 'dbe8968f75c34d9eb3d3c4c226aa8fdf', in update group 'default'
+Aug 06 14:50:39 tutorial zincati[889]: [INFO ] initialization complete, auto-updates logic enabled
+----
+
+One other interesting thing to do is view the logs from Ignition in case there is anything interesting there you may want to investigate:
+
+[source,bash]
+----
+[core@tutorial ~]$ journalctl -t ignition | cat
+...
+----
+
+And finally, of course you can use the podman (or Docker) command to inspect the current state of containers on the system:
+
+[source,bash]
+----
+[core@tutorial ~]$ podman version
+[core@tutorial ~]$ podman info
+----
+
+*Note:* `podman` commands can be run as root or as non-root user. However you will need use `sudo` for `docker` commands as the core user is not part of the `docker` group by default.
```

Comment:
```suggestion
NOTE: `podman` commands can be run as root or as non-root user. `docker` commands need to be run as root via `sudo` unless the user has been added to the `docker` group.
```

Asciidoc picks up this and makes a nicely formatted box for it if it starts with `NOTE:`